src/index.ts

```diff
@@ -69,14 +70,15 @@ export default function(settings?: ProblemMwSettings): Middleware {
         ctx.response.body.detail = detail;
       }
 
-      if (clientError) {
-        // tslint:disable-next-line no-console
-        console.warn(e);
-      } else {
-        // tslint:disable-next-line no-console
-        console.error(e);
+      if (settings) {
```

Comment:
Could you make a `quiet` variable and move it to the top of the file (around where `debug` is also defined?)
A second issue is that this will now only emit any logs _if_ settings is defined, but I think this problem can fixed easier if we just use 1 quiet flag.